zerver/models/presence.py

```diff
@@ -70,6 +76,11 @@ def status_from_string(status: str) -> Optional[int]:
         return None
 
 
+class PresenceSequence(models.Model):
+    realm = models.OneToOneField(Realm, on_delete=CASCADE)
+    last_update_id = models.PositiveBigIntegerField()
```

Comment:
Should this have an explicit `default=0`?

Reply:
No strong preference, but I felt that code that creates `PresenceSequence`s should be mindful of what it's doing and set the value explicitly, so I left this without a `default`. We can add it easily too if that seems better though.

Reply:
Yeah, OK, that reasoning seems fair.